polyaxon/db/migrations/0001_initial.py

```diff
@@ -578,7 +578,7 @@ class Migration(migrations.Migration):
                 ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                 ('created_at', models.DateTimeField(auto_now_add=True, db_index=True)),
                 ('updated_at', models.DateTimeField(auto_now=True)),
-                ('provider', models.CharField(choices=[('github', 'github'), ('bitbucket', 'bitbucket'), ('gitlab', 'gitlab')], max_length=32)),
+                ('provider', models.CharField(choices=[('github', 'github'), ('bitbucket', 'bitbucket'), ('gitlab', 'gitlab'), ('azure', 'azure')], max_length=32)),
```

Comment:
I think we should create a new migration, to allow people to upgrade their cluster with breaking changes.

You should revert the change in this file, and run the command:

```
cmd/manage makemigrations
```